### PR TITLE
Governance Model Document Creation

### PR DESCRIPTION
**Pull request by ewferg**
* **Tickets addressed:** None
* **Review:** by commit
* **Merge strategy:** Merge (no squash)  

## Description
Initial creation of an Aerie open source governance model. This simply adds a GOVERNANCE.md document to describe our governance.

### CODE REVIEW

**Comment by ewferg** [comment]
There is still some additional things I want to add prior to merging this in (e.g. listing of collaborators, TSC, and PMC), but I wanted to get this out there for an initial review.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+The Aerie project aims to create an open source community that encourages transparency, contributions, and collaboration, but maintains sound technical and quality standards. Our goal is to build a community comprised of members across the the space community and beyond, including from private organizations, universities, Federaly Funded Research and Development Ceneters (FRDCs), other government organizations, and international organizations. 
+
+The project follows a fairly liberal contribution model where people and/or organizations who do the most work will have the most influence on project direction. Technical decision making will primarily be made through a "[consensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)" approach. 
+
+## Roles
+
+### User
+
+Anyone who has downloaded, deployed, or operated the Aerie system to meet a specific objective. Aeire was primarily designed for space mission activity planning, modeling, and simulation, but let us know if you've found other uses for it.
```
Comment (dyst5422) [inline GOVERNANCE.md]: Typo Aeire

**Comment by camargo** [comment]
This looks good to me. Please run spell check over the document and then I think we can merge.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+### Contributor
+
+Contributors include anyone that provides input to the project. This includes code, documentation, graphics, designs, or anything else that tangibly improves the project. We encourage you to start contributing right away by joining our [Discussions](https://github.com/NASA-AMMOS/aerie/discussions) or submitting an [Issue](https://github.com/NASA-AMMOS/aerie/issues). 
+ 
+### Collaborator
+
+Subset of contributors who have been given write access to one or more of the Aerie repositories. Both contributors and collaborators can propose changes to the project via pull requests, but only collaborators can formally review and approve (merge) these requests. Any contributor who has made a non-trivial contribution should be on-boarded as a collaborator in a timely manner. 
+
+If you are planning on making a substantial contribution to the project or feel as though you should be given write access to a repository, please reach out to TBD (some email list??)...
```
Comment (Mythicaeda) [inline GOVERNANCE.md]: Flagging lines that have TBDs so we don't accidentally merge before they're filled in.

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+A subset of the collaborators forms the Technical Steering Committee (TSC). The TSC has authority over the following aspects of this project:
+
+- Technical direction and guidance
+- Project governance and process 
+- Contribution policy
+- Conduct guidelines
+- Maintaining the list of collaborators
+
+The current list of TSC members is maintained TBD
```
Comment (Mythicaeda) [inline GOVERNANCE.md]: Flagging lines that have TBDs so we don't accidentally merge before they're filled in.

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+The Project Management Committee (PMC) is made up of sponsor organization representives (i.e. those providing funding to the project) and key stakeholders who rely or plan to rely on Aerie to meet a crticial need (e.g. project using Aerie for space mission operations). The PMC has the following primary responsibilities
+
+- Maintaining the overall project roadmap
+- Determining project requirements and committments to sponsors and stakeholders
+- Assessing available resources and allocating them across the project
+- Monitoring and reporting on progress against the roadmap 
+- On-boarding new sponsors and stakeholders
+- Addressing any legal considerations
+
+The current list of PMC members is maintained TBD. If your project or organization is planning to use Aerie and you would like to join the PMC, please contact TBD....
```
Comment (Mythicaeda) [inline GOVERNANCE.md]: Two TBDs in this line.

File: GOVERNANCE.md (hunk 1 above)
Comment (ewferg) [reply GOVERNANCE.md]: Fixed!

**Review by Mythicaeda — APPROVED** [review]
Once the TBDs are filled in, this looks good to go!

File: GOVERNANCE.md (hunk 2 above)
Comment (camargo) [reply GOVERNANCE.md]: @ewferg I think we can use [aerie_support@jpl.nasa.gov](mailto:aerie_support@jpl.nasa.gov) for now (same one we use for our [SECURITY.md](https://github.com/NASA-AMMOS/aerie/blob/develop/SECURITY.md) and [CODE_OF_CONDUCT.md](https://github.com/NASA-AMMOS/aerie/blob/develop/CODE_OF_CONDUCT.md) documents)

File: GOVERNANCE.md (hunk 3 above)
Comment (camargo) [reply GOVERNANCE.md]: I think this is okay to merge since we don't actually have these yet. I'll defer to @ewferg tho.

Comment (Mythicaeda) [reply GOVERNANCE.md]: Regarding that deferral, if you agree that it's okay to merge, @ewferg, then my feedback regarding the TBDs changes to a suggestion that the lines with 'TBD' be reworded to flow better, as the phrase `is maintained To Be Determined` was why I thought they were details that needed to be filled in before merging.

Comment (ewferg) [reply GOVERNANCE.md]: I would like to go ahead and knock out the TBDs before we merge this in, hopefully in the next day or two.

File: GOVERNANCE.md (hunk 2 above)
Comment (ewferg) [reply GOVERNANCE.md]: Ok, that sounds good to me for now. Out of curiosity, do you know if we can add non-JPLers to that email list?

File: GOVERNANCE.md (hunk 4 above)
Comment (ewferg) [reply GOVERNANCE.md]: I went ahead and removed the TBDs although the exact PMC members are still in work. I am also debating where to put the PMC list. I am thinking either inline here or in the README. Any preferences?

File: GOVERNANCE.md (hunk 3 above)
Comment (ewferg) [reply GOVERNANCE.md]: Went ahead and added a proposed TSC list.

File: GOVERNANCE.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+- Project governance and process 
+- Contribution policy
+- Conduct guidelines
+- Maintaining the list of collaborators
+
+### TSC Committee Members
+- Chris Camargo ([camargo](https://github.com/camargo)), Jet Propulsion Laboratory
+- Matt Dailis ([mattdailis](https://github.com/mattdailis)), Jet Propulsion Laboratory
+- Jonathan Castello ([Twisol](https://github.com/Twisol)), University of California, Santa Cruz
+- Basak Alper Ramaswamy, Jet Propulsion Laboratory
```
Comment (mattdailis) [inline GOVERNANCE.md]:
```suggestion
- Basak Alper Ramaswamy ([bashbash](https://github.com/bashbash)), Jet Propulsion Laboratory

```

File: GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+However, the TSC also has responsibility over some projects, which are dependencies of the core projects:
+
+- https://github.com/NASA-AMMOS/aerie-docs
+- https://github.com/NASA-AMMOS/aerie-gateway
+- https://github.com/NASA-AMMOS/aerie-ts-user-code-runner
+- https://github.com/NASA-AMMOS/aerie-ampcs
+
+Addtionally, the TSC has authority over a few other "Aerie Extended Universe" projects:
+
+- https://github.com/NASA-AMMOS/aerie-cli
```
Comment (mattdailis) [inline GOVERNANCE.md:65]: JFYI, aerie-cli is still private

**Review by mattdailis — APPROVED** [review]

File: GOVERNANCE.md (hunk 6 above)
Comment (ewferg) [reply GOVERNANCE.md:65]: I am waiting for the thumbs up from Parker before we go live with this one.

File: GOVERNANCE.md (hunk 2 above)
Comment (ewferg) [reply GOVERNANCE.md]: @camargo, I'll mark this as resolved, but let me know about the email list when you get a chance.

**Pull request merged**